shell/platform/tizen/channels/settings_channel_linux.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "settings_channel.h"
+
+#include <chrono>
+#include <ctime>
+
+namespace flutter {
+
+void SettingsChannel::Init() {
+  std::locale::global(std::locale(""));
+}
+
+void SettingsChannel::Dispose() {}
+
+bool SettingsChannel::GetSettingValueOf24Format(bool* value) {
+  // This is a temporary implementation because I didn't know how to properly implement it.
+  // I believe someone will replace it with a better implementation.
+  // This function operates normally only in the following locale.
+  // en_US.UTF-8, ko_KR.UTF-8
```

Comment:
I think always returning false is better than producing inconsistent results.

Reply:
I was trying to match the tizen side, but ... you're right.